t/26-video_stream.t

```diff
@@ -266,7 +281,7 @@ subtest 'v4l2 resolution' => sub {
     %v4l2_ctl_results = (
         '--query-dv-timings' => '640x480p60',
     );
-    $console->{dv_timings_last_check} = time - 4;
+    $console->{dv_timings_last_check} = $console->DV_TIMINGS_CHECK_INTERVAL;
```

Comment:
I would avoid invoking constants like this. It works but is not really how they are intended to be used.

```suggestion
    $console->{dv_timings_last_check} = consoles::video_stream::DV_TIMINGS_CHECK_INTERVAL;
```

The same counts for the other occurrence.